single-function/longest-common-prefix/longest-common-prefix.test.js

```diff
@@ -0,0 +1,30 @@
+const longestCommonPrefix = require('./02-impl-revision');
+
+describe('addIntegerLists', () => {
+  it('given empty array, [] return ""', () => {
+
+    const arrayOfStrings = [];
+
+    const expected = '';
+
+    expect(longestCommonPrefix(arrayOfStrings)).toEqual(expected);
+  });
+
+  it('given array, ["aaabb", "aabb", "xaacc"] return ""', () => {
+
+    const arrayOfStrings = ["aaabb", "aabb", "xaacc"];
```

Comment:
single quotes used elsewhere in the code, let's keep it standardized on single quotes.